lib/auth/clt.go

```diff
@@ -860,7 +860,7 @@ func (c *Client) DeleteWebSession(ctx context.Context, user string, sid string)
 	return trace.Wrap(err)
 }
 
-// GenerateHostCert takes the public key in the Open SSH ``authorized_keys``
+// GenerateHostCert takes the public key in the Open SSH “authorized_keys“
```

Comment:
Still in nitpick mode, but kinda unusual to add the weird unicode quotes here (and 2x left double quote character at that). Maybe just regular double quotes?

```suggestion
// GenerateHostCert takes the public key in the Open SSH "authorized_keys"
```

Reply:
gofmt doing weird unicode replacements